### PR TITLE
RESTWS-871: Follow naming convention for constants

### PR DESCRIPTION
**Pull request by docjon09**
## Description of what I changed

Changed names of constant variables to follow the naming convention

## Issue I worked on

see https://issues.openmrs.org/browse/RESTWS-871

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: integration-tests/src/test/java/org/openmrs/module/webservices/rest/ITBase.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 public abstract class ITBase {
 	
-	private static final Object serverStartupLock = new Object();
+	private static final Object SERVER_STARTUP_LOCK = new Object();
```
Comment (kdaud) [inline integration-tests/src/test/java/org/openmrs/module/webservices/rest/ITBase.java:30]: @docjon09 are you sure `serverStartupLock` is a variable?

Comment (docjon09) [reply integration-tests/src/test/java/org/openmrs/module/webservices/rest/ITBase.java]: I didn't knew objects aren't considered variables. I have updated it now. Thanks.

**Comment by kdaud** [comment]
Is the ticket url correct?

**Comment by docjon09** [comment]
> Is the ticket url correct?

yes?

**Comment by docjon09** [comment]
> Have you tried to check it out and confirmed based on the work in this PR?

I have updated the ticket link now.

**Comment by docjon09** [comment]
> What about the commit message?
I can't change it now, right?

**Comment by kdaud** [comment]
What about editing it to a preferred commit message?

**Comment by kdaud** [comment]
Isn't there any variable that has been left out?

**Comment by docjon09** [comment]
> Isn't there any variable that has been left out?

I have capitalized the names of all variables with keyword 'static final'. But I have left the 'final' variables unchanged as some websites say that they need not be capitalized.

**Comment by kdaud** [comment]
> I have left the 'final' variables unchanged as some websites say that they need not be capitalized.

Could you share the link to the documentation?

**Comment by kdaud** [comment]
>  But I wasn't sure whether final fields are considered constants

Is this link of help? https://www.thoughtco.com/constant-2034049#:~:text=A%20constant%20is%20a%20variable,read%20and%20understood%20by%20others.

**Comment by jayasanka-sack** [comment]
Hey @docjon09, I noticed that there are some pending changes in this PR. Are there any updates on when it will be completed? Let me know if there's anything I can do to help move it forward.

**Comment by docjon09** [comment]
> Hey @docjon09, I noticed that there are some pending changes in this PR. Are there any updates on when it will be completed? Let me know if there's anything I can do to help move it forward.

Sorry, I forgot about this.
Yes, I require help here. According to [https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html](url), all class constants should be capitalized. But I do not know whether 'final' variables are considered class constants as their values are different for different class instatiations.
Also, here [https://softwareengineering.stackexchange.com/a/252252](url), it says that constants named serialVersionUID and serialPersistentFields need not be capitalized. These names are there in this repo and have not been capitalized.

**Comment by jayasanka-sack** [comment]
Hi @docjon09, Thank you for bringing this to our attention. You are correct that according to the Oracle Java Code Conventions, all class constants should be capitalized, but for instance final variables that are not static, Oracle does not have any specific guidelines on whether to use uppercase or lowercase letters.

Let's get this PR merged first. We apologize for the confusion and any inconsistencies in our naming conventions. I will make sure to review and update our docs accordingly to ensure consistency and compliance with industry standards.  Thanks again for bringing this to our attention. ❤️

**Review by jayasanka-sack — APPROVED** [review]

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/wrapper/openmrs1_8/CohortMember1_8.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
  */
 public class CohortMember1_8 implements Serializable {
 	
-	public static final long serialVersionUID = 1L;
+	public static final long SERIAL_VERSION_UID = 1L;
```
Comment (jayasanka-sack) [inline omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/wrapper/openmrs1_8/CohortMember1_8.java:25]: Btw, you should not use uppercase for `serialVersionUID`. Even though constants should be capitalize, there is an exception for the variable `serialVersionUID`, which is a special variable that is used by the serialization mechanism. Same applied for `serialPersistentFields` valriable.

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Pull request merged**